cornflow/endpoints/execution.py

```diff
@@ -60,26 +59,33 @@ def post(self):
           the reference_id for the newly created execution if successful) and a integer wit the HTTP status code
         :rtype: Tuple(dict, integer)
         """
+        config = current_app.config
+        airflow_conf = dict(url=config['AIRFLOW_URL'], user=config['AIRFLOW_USER'], pwd=config['AIRFLOW_PWD'])
+
         self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
         result = self.post_list(request)
 
         not_run = request.args.get('run', '1') == '0'
 
+        # if we failed to save the execution, we do not continue
         if result[1] >= 300 or not_run:
             return result
-        # To send the absolute url:
-        # url_for(endpoint_name, _external=True)
-        airflow_client = Airflow(current_app.config['AIRFLOW_URL'],
-                                 current_app.config['AIRFLOW_USER'],
-                                 current_app.config['AIRFLOW_PWD'])
-        if not airflow_client.is_alive():
+
+        # We now try to launch the task in airflow
+        af_client = Airflow(**airflow_conf)
+        if not af_client.is_alive():
             return {'error': "Airflow is not accessible"}, 400
         execution_id = result[0][self.primary_key]
         try:
-            airflow_client.run_dag(execution_id, current_app.config['CORNFLOW_URL'])
+            response = af_client.run_dag(execution_id, config['CORNFLOW_URL'], dag_name='solve_model_dag')
         except AirflowApiError as err:
```

Comment:
Same here as the above message

Reply:
yes, this is maybe too technical. Useful for debugging but we should change it for something less cryptic. Or maybe put both (the airflow error but first introduce something about the "scheduler")

Reply:
yes, this is maybe too technical. Useful for debugging but we should change it for something less cryptic. Or maybe put both (the airflow error but first introduce something about the "schedul